### ISSUE TITLE
Stuck in a Kactus failing loop

### ISSUE THREAD

**Issue by xuv** [kactus-io/kactus#103]
## Description

When Kactus fails and crashes, it proposes to quit and restart. If the failing keeps happening, there is no option offered to get out of a `quit and restart` loop.

## Version

* Kactus: 0.3.6
* Operating system: Mac OS 10:13.4

## Additional Information

The error where it fails:
```
TypeError: Cannot read property 'mediaType' of undefined
    at render (app/src/ui/diff/image-diffs/image-container.tsx:19:38)
    at g (app/node_modules/react-dom/cjs/react-dom.production.min.js:130:323)
    at beginWork (app/node_modules/react-dom/cjs/react-dom.production.min.js:137:236)
    at a (app/node_modules/react-dom/cjs/react-dom.production.min.js:175:99)
    at e (app/node_modules/react-dom/cjs/react-dom.production.min.js:175:376)
    at M (app/node_modules/react-dom/cjs/react-dom.production.min.js:181:155)
    at n (app/node_modules/react-dom/cjs/react-dom.production.min.js:180:377)
    at A (app/node_modules/react-dom/cjs/react-dom.production.min.js:180:221)
    at _ (app/node_modules/react-dom/cjs/react-dom.production.min.js:179:80)
    at h (app/node_modules/react-dom/cjs/react-dom.production.min.js:178:34)

Version: 0.3.6
OS: Mac OS 10.13.4
```

A screenshot of the particular window with only a `quit and relaunch` button.
![screen shot 2018-06-05 at 12 06 23](https://user-images.githubusercontent.com/192539/40988618-8bfd835a-68b9-11e8-8d4c-14b5012da562.png)

**Comment by mathieudutour** [comment]
should be fixed in 0.3.7

**Comment by xuv** [comment]
Hello @mathieudutour 

Thanks for fixing the TypeError, but I don't think this issue is fixed.
The problem here is when Kactus crashes, it is impossible to get out of a crash-restart loop because the pattern is to automatically relaunch Kactus after a crash.

Since 0.3.7 has an issue when starting #104, the crash-restart loop is happening right from the start.

This issue is asking for a UI button to add the option to just quit Kactus without relaunching it in case of crash. 

Unfortunately an Mac OS force quit or `killall Kactus` does not help either.
